packages/server/bin.js

```diff
@@ -20,7 +20,8 @@ server(
       'https://serge-dev.herokuapp.com',
       'http://localhost:3000',
       'http://localhost:8080',
-      'http://localhost:8000'
+      'http://localhost:8000',
+      'http://localhost:6611'
```

Comment:
Can you explain why we've registered this port?

Oh, and do you know why we have `main.js` _and_ `bin.js` ?

I know you didn't create them - I'm just wondering if you understand why.

Reply:
This port is register to get out from CORS issues while request from other apps like storybook and web app.

Sorry, no idea about the difference between that file but I just see we had added this URL in two files so I added it for reference.

It would be good if you give more explanations here if I had added wrongly :)

Reply:
Aah, it's the port StoryBook is registered to use:
![image](https://user-images.githubusercontent.com/1108513/115392169-2888e480-a1d8-11eb-865a-5499505bec33.png)

Could you also do some digging (file-searching) to see where the two files are used, please?

We can then put explanatory comments in the files themselves.

Reply:
ok